awx_collection/plugins/modules/tower_project.py

```diff
@@ -97,6 +97,11 @@
       default: "present"
       choices: ["present", "absent"]
       type: str
+    wait:
+      description:
+        - Wait for project to sync before running the associated job template.
```

Comment:
This description might be a little unclear. I think job templates are the _reason_ you use this, but this wording suggests something out of place. You can use multiple `-`, so I would suggest something like

 - Wait for project to finish update before returning
 - Useful to assure playbook files are populated so that job templates that rely on the project can be created

that still sounds a little rough maybe?

Reply:
@AlanCoding, what do you think of:

- Provides option (True by default) to wait for completed project sync before returning
- Assures playbook files are populated so that job templates that rely on the project can be successfully created

Reply:
yeah that's a much better revision, I would go with that